### PR TITLE
LPS-142890

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related Issue: https://issues.liferay.com/browse/LPS-142890

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
FYI, @natocesarrego, there's a frontend commit, but it was made in pair programming with @miltonmc, so it has already passed his review.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContextTest.java [hunk 1]
```diff
@@ -160,6 +163,47 @@ public void testGetSharedFormURLForSharedFormInstance() throws Exception {
...
+		);
+
+		when(
+			ddmFormInstanceSettings.neverExpire()
+		).thenReturn(
+			false
+		);
+
+		Assert.assertTrue(_ddmFormAdminDisplayContext.isFormExpired());
+	}
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContextTest.java:205]: @rodrigopaulino, I see that this unit test checks 3 test cases, in light of this how about split it into 3 unit tests? e.g. a unit test for each test case. For example:

- testFormNotExpiredWithInexistentForm
- testFormNotExpiredWithNeverExpireSetting
- testFormExpired

It's just a suggestion. :slightly_smiling_face: Feel free to use it or not. :wink:

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:stop

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
Working on another pull that will contain this one.

**Comment by rodrigopaulino** [comment]
ci:close

**Comment by rodrigopaulino** [comment]
Resent https://github.com/liferay-forms/liferay-portal/pull/1183

**Pull request closed without merging**